Fixed compatibility with Zope 4.5.2 by making sure Location header is string.
On Python 2 it could be unicode for the users and groups end points.
Fixes `issue 1019 <https://github.com/plone/plone.restapi/issues/1019>`_.
[maurits]